### ISSUE TITLE
Exception on adding a new user

### ISSUE THREAD

**Issue by liffeyd** [dnnsoftware/Dnn.Platform#3441]
## Description of bug
Addting a new user results in an exception. The user is added but an exception is thrown. 

## Steps to reproduce
PersonaBar > Users > Add User
Fill in the required details and click "Save"

## Current behavior
Exception thrown - details below.
The user is added but no email is sent and the exception below is written to the error log.

An email server is setup and a test email is sent and received successfully.

## Expected behavior
User is added without exception.

## Screenshots
![AddUserError_1](https://user-images.githubusercontent.com/7514238/71617408-d7a04780-2bb2-11ea-954f-3484ba65b948.png)

DNN Version
![AddUserError_2](https://user-images.githubusercontent.com/7514238/71617660-f5ba7780-2bb3-11ea-884a-4c67b7100982.png)

Updated from 8.0.4
![AddUserError_3](https://user-images.githubusercontent.com/7514238/71617711-30bcab00-2bb4-11ea-8865-f903f1268644.png)

## Error information
Dnn.PersonaBar.Users.Services.UsersController

System.NullReferenceException: Object reference not set to an instance of an object.
   at DotNetNuke.Services.Mail.Mail.SendMail(UserInfo user, MessageType msgType, PortalSettings settings)
   at DotNetNuke.Entities.Users.UserRegistrationEmailNotifier.NotifyAdministrator(UserInfo user)
   at DotNetNuke.Entities.Users.UserEventHandlers.UserCreated(Object sender, UserEventArgs args)
   at DotNetNuke.Entities.EventManager.OnUserCreated(UserEventArgs args)
   at DotNetNuke.Entities.Users.UserController.CreateUser(UserInfo& user, Boolean sendEmailNotification)
   at Dnn.PersonaBar.Users.Components.RegisterController.Register(RegisterationDetails registerationDetails)
   at Dnn.PersonaBar.Users.Services.UsersController.CreateUser(CreateUserContract contract)

## Additional context
Provide any additional context that may be helpful in understanding and/or resolving the bug.

## Affected version

* [ ] 10.0.0 alpha build
* [ ] 9.5.0 alpha build
* [X ] 9.4.4 latest supported release

## Affected browser

ALL

**Comment by sleupold** [comment]
- has SMTP properly been onfigured?
- is this a new or upgraded website?
- does this happen for any user (or just users with a . in the name)?

**Comment by liffeyd** [comment]
Sebastian,
As per above, SMTP appears to be configured correctly. A test worked. My email server logs show the test message from the site. On adding the user no message is received by the mail server.

The user is added correctly and can login successfully.

The same occurrs with or without a "." in the username.

**Comment by sleupold** [comment]
@liffeyd 
according to the stack trace, the error occurs while sending the mail (NotifyAdministrator(UserInfo user)). Does the site admin (specified in site settings) have a valid email address and displayname?

**Comment by liffeyd** [comment]
Sebastian,
Interestingly, on a fresh DNN 9.4.4 install, with no SMTP server details entered, the User is added without error. 

I then removed the SMTP settings, from the problem site,  and tried adding a user with the same result.

I have deleted bin and copied from the fresh install and the error persists.

As this is not something anyone else seems to have reported, and the site is only a development instance, I will bin it and continue with the fresh instance.

Thanks for your input :)
